src/http/api/resources/files-regular.js

```diff
@@ -95,7 +95,9 @@ exports.get = {
       async function * (source) {
         for await (const file of source) {
           const header = {
-            name: file.path
+            name: file.path,
+            mtime: file.mtime ? new Date(file.mtime.secs * 1000) : null,
```

Comment:
Similar to my comment on https://github.com/ipfs/js-ipfs-http-client/pull/1235 - can this support the `nsecs` mtime component too?

Reply:
I think this might be more complicated than I'd previously thought - `it-tar` emits regular tar files, right?  As I understand it, regular tar files only support seconds for mtimes, higher resolution requires GNU Tar files so `it-tar` may need to be updated to support producing these.

Reply:
Correct https://github.com/alanshaw/it-tar/blob/master/pack-headers.js#L96